### PR TITLE
Removing the cctriples class.

### PR DESCRIPTION
**Pull request by lothian**
## Description
This PR eliminates the unnecessary `cctriples` class in favor of using `cctriples.py` as a collection of functions.  This simplifies the interface a bit and allows `ccwfn` and other classes to access the capabilities of `cctriples.py` without instantiating a new object internally.  

## Todos
  - [X] Remove the class structure of `cctriples`.
  - [X] Adjust `ccwfn` to compute (T) and triples contributions to CC3 through direct function calls.
  - [X] Adjust test 005 for the changes.  (Test 031 remains unchanged.)

## Status
- [X] Ready to go

### CODE REVIEW

**Comment by zw0930** [comment]
The modification looks good to me. For PyTorch, this change won't affect the existing PyTorch-specific cases. Some changes may need to be made for running CC3 on GPU that was not included in PR#50. But those ones are not related to the change in this PR. Do you think it would be better to leave the last Todos to another PR afterwards if needed?

**Comment by lothian** [comment]
> The modification looks good to me. For PyTorch, this change won't affect the existing PyTorch-specific cases. Some changes may need to be made for running CC3 on GPU that was not included in PR#50. But those ones are not related to the change in this PR. Do you think it would be better to leave the last Todos to another PR afterwards if needed?

I was working to get PyTorch working on my M1 Mac, but after much work to get the correct environment set up, it turns out that the Mac MPS GPU doesn't have the ComplexDouble dtype! 

As you suggested, I'm going to drop the PyTorch part of the PR for later updates so that I can move forward with other PRs.

**Comment by zw0930** [comment]
> Mac MPS GPU

That's a bummer. Just looked it up and I saw PyTorch 1.12 only support complex32. Hope they will include at least complex64 soon. It would be cool to be able to run some single-precision calculations on your Mac.

**Review by zw0930 — APPROVED** [review]

**Review by bgpeyton — APPROVED** [review]
I don't want to be a hold-up any longer on this - documentation can (and should!) always be added later. 😃

File: pycc/ccwfn.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 from .utils import helper_diis, cc_contract
 from .hamiltonian import Hamiltonian
 from .local import Local
-from .cctriples import cctriples 
+from .cctriples import t_tjl, t3c_ijk
```
[the remark this answers is not in the document]
Comment (lothian) [reply pycc/ccwfn.py:16]: The Lee and Rendell formulation is the most efficient in terms of CPU requirements.

**Pull request merged**